compute/cloud-client/src/test/java/compute/disks/HyperdisksIT.java

```diff
@@ -73,9 +73,9 @@ public static void cleanup()
        throws IOException, InterruptedException, ExecutionException, TimeoutException {
     // Delete all disks created for testing.
     DeleteDisk.deleteDisk(PROJECT_ID, ZONE, HYPERDISK_NAME);
-    //DeleteDisk.deleteDisk(PROJECT_ID, ZONE, HYPERDISK_IN_POOL_NAME);
+    DeleteDisk.deleteDisk(PROJECT_ID, ZONE, HYPERDISK_IN_POOL_NAME);
```

Comment:
According to my understanding the creation of the hyperdisk storage pool does not create disks. Please, remove unnecessary invocations.

Reply:
As I understand, HYPERDISK_IN_POOL_NAME - it's disk created in the third test and it should be deleted after testing